packages/schematics/angular/universal/files/__sourceDir__/__appDir__/__rootModuleFileName__

```diff
@@ -7,7 +7,7 @@ import { AppComponent } from './app.component';
 @NgModule({
   imports: [
     AppModule,
-    ServerModule,
+    ServerModule
```

Comment:
Why?

Reply:
The adding of imports logic does not support dealing with trailing commas being present yet.

Reply:
that's a huge issue. Is this a problem only with this new logic?